paimon-core/src/test/java/org/apache/paimon/iceberg/IcebergCompatibilityTest.java

```diff
@@ -271,6 +274,101 @@ public void testAppendOnlyTableWithAllTypes() throws Exception {
                 r -> "");
     }
 
+    @Test
+    public void testPartitionedPrimaryKeyTable_Timestamp() throws Exception {
```

Comment:
No need to create a new test method. Just add a few lines to `testAppendOnlyTableWithAllTypes`. Also don't use underline, use camel case.

Reply:
![image](https://github.com/user-attachments/assets/f16feb7e-789f-4db6-b51d-3aa5d01be8ac)
Using the default method results in an error，for this code,beacause icerberg avor fomart not support  logical type(LogicalTypes$TimeMillis)
![image](https://github.com/user-attachments/assets/17286dbc-7c28-484d-b3c5-6f0c0e8c1b61)

so, i remove this code ,and create new  testmethod.

if  i fix this bug?  let adpot LogicalTypes$TimeMillis ?

Reply:
I suggest that you only test on `TIMESTAMP(6)` Paimon type.

Reply:
Thank you very much, I fixed the problem and passed the test
and test only on TIMESTAMP(6)